runtime/gc_check/CheckCycle.cpp

```diff
@@ -429,6 +429,13 @@ GC_CheckCycle::initialize(const char *options)
 void
 GC_CheckCycle::run(GCCheckInvokedBy invokedBy, UDATA filterFlags)
 {
+	UDATA originalMiscFlags = _miscFlags;
+	/* set the midscavenge miscFlags if is in concurrent scavenger to check forwarded pointers */
+	if (((MM_GCExtensions*)_javaVM->gcExtensions)->isConcurrentScavengerInProgress()) {
+		_miscFlags |= J9MODRON_GCCHK_MISC_MIDSCAVENGE;
```

Comment:
please declare `extensions` at the separate line, like:
```
	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(_javaVM);
```
Also please fix first line in GC_CheckCycle::initialize() to use getExtensions() as well.
Also please replace
```
#include "GCExtensionsBase.hpp"
```
to
```
#include "GCExtensions.hpp"
```